packages/eslint-plugin-pf-codemods/test/rules/v5/warn-event-api.js

```diff
@@ -0,0 +1,86 @@
+const ruleTester = require('../../ruletester');
+const rule = require('../../../lib/rules/v5/warn-event-api');
+
+ruleTester.run("warn-event-api", rule, {
+  valid: [
```

Comment:
not really necessary as the rule is only on import declarations.. could test `import { DropdownToggle } from '@foo/bar'` but no biggie

Reply:
another option would be to put the warnings on the jsx elements like here https://github.com/patternfly/pf-codemods/pull/267

this would point them in the code closer to where they'd want to make changes, but again, not a blocker